scalyr_agent/monitor_utils/k8s.py

```diff
@@ -80,12 +119,24 @@ def query_api( self, path, pretty=0 ):
 
     def query_pod( self, namespace, pod ):
         """Wrapper to query a pod in a namespace"""
-        return self.query_api( '/api/v1/namespaces/%s/pods/%s' % (namespace, pod) )
+        if not pod or not namespace:
```

Comment:
Is this an error?  If they pass in an empty `pod` or `namespace`, should we just thrown an exception?